ClientTests/TestFavicons.swift

```diff
@@ -21,30 +21,30 @@ class TestFavicons: ProfileTest {
         self.waitForExpectations(timeout: 100, handler: nil)
     }
 
-    func testFaviconFetcherParse() {
-        let expectation = self.expectation(description: "Wait for Favicons to be fetched")
-
-        let profile = MockProfile()
-        // I want a site that also has an iOS app so I can get "apple-touch-icon-precomposed" icons as well
-        let url = URL(string: "https://instagram.com")
-        FaviconFetcher.getForURL(url!, profile: profile).uponQueue(.main) { result in
-            guard let favicons = result.successValue, favicons.count > 0, let url = favicons.first?.url.asURL else {
-                XCTFail("Favicons were not found.")
-                return expectation.fulfill()
-            }
-            XCTAssertEqual(favicons.count, 1, "Instagram should have a Favicon.")
-
-            WebImageCacheWithNoPrivacyProtectionManager.shared.load(from: url) { image, _, _, _, _ in
-                guard let image = image else {
-                    XCTFail("Not a valid URL provided for a favicon.")
-                    return expectation.fulfill()
-                }
-                XCTAssertNotEqual(image.size, .zero)
-                expectation.fulfill()
-            }
-        }
-        self.waitForExpectations(timeout: 3000, handler: nil)
-    }
+//    func testFaviconFetcherParse() {
```

Comment:
Delete unused tests complete, and add ticket to re-add favicon tests.

Reply:
Done (#393)